modules/haberdasher-labs_hat-controlled-module_v0.1.0.json

```diff
@@ -0,0 +1,304 @@
+{
+  "id": "haberdasher-labs_hat-controlled-module",
+  "version": "v0.1.0",
+  "name": "Hat Controlled Module",
+  "details": [
+    "Enables an authorized controller hat to set wearer status and/or hat activation status for other hat(s).",
+    "Unlike Passthrough Module, this module will work even when chained together with other modules."
+  ],
+  "links": [
+    {
+      "label": "GitHub",
+      "link": "https://github.com/Hats-Protocol/passthrough-modules"
+    }
+  ],
+  "parameters": [
+    {
+      "label": "Controller Hat",
+      "functionName": "CONTROLLER_HAT",
+      "displayType": "hat"
+    }
+  ],
+  "type": {
+    "eligibility": true,
+    "toggle": true,
+    "hatter": false
+  },
+  "tags": [],
+  "implementationAddress": "0x94702C4337BB5BF59A2B6c4E0187333b180d153B",
+  "deployments": [
+    {
+      "chainId": "11155111",
+      "block": "6686024"
+    },
+    {
+      "chainId": "10",
+      "block": "125364801"
+    },
+    {
+      "chainId": "137",
+      "block": "61819779"
+    },
+    {
+      "chainId": "42161",
+      "block": "253476246"
+    },
+    {
+      "chainId": "100",
+      "block": "35999017"
+    },
+    {
+      "chainId": "8453",
+      "block": "19769801"
+    },
+    {
+      "chainId": "42220",
+      "block": "27731366"
+    }
+  ],
+  "creationArgs": {
+    "useHatId": true,
+    "immutable": [
+      {
+        "name": "Controller Hat",
+        "description": "The hat that will control wearer eligibility and/or activation status of hats controlled by this module",
+        "type": "uint256",
+        "example": "53919893334301279589334030174039261347274288845081144962207220498432",
+        "displayType": "hat"
+      }
+    ],
+    "mutable": []
+  },
+  "customRoles": [
+    {
+      "id": "controller",
+      "name": "Eligibility/Toggle Controller",
+      "criteria": "CONTROLLER_HAT"
+    }
+  ],
+  "writeFunctions": [
+    {
+      "roles": ["controller"],
+      "functionName": "setWearerStatus",
+      "label": "Set Wearer's Status",
+      "description": "Set the eligibility and standing of a chosen hat wearer",
+      "primary": true,
+      "args": [
+        {
+          "name": "Hat",
+          "description": "ID of the hat to set the wearer status for",
+          "type": "uint256",
+          "displayType": "hat"
+        },
+        {
+          "name": "Wearer",
+          "description": "The address to set the wearer status for",
+          "type": "address",
+          "displayType": "default"
+        },
```

Comment:
Need to switch the order between the `wearer` and `hat` args to match the contract function